common/components/BalanceSidebar/TokenBalances.jsx

```diff
@@ -0,0 +1,78 @@
+// @flow
+import React from 'react';
+import translate from 'translations';
+import TokenRow from './TokenRow';
+import AddCustomTokenForm from './AddCustomTokenForm';
+import type { TokenBalance } from 'selectors/wallet';
+import type { Token } from 'config/data';
+
+export default class TokenBalances extends React.Component {
+  props: {
+    tokens: TokenBalance[],
+    onAddCustomToken: (token: Token) => any,
+    onRemoveCustomToken: (symbol: string) => any
+  };
+  state = {
+    showAllTokens: false,
+    showCustomTokenForm: false
+  };
+
+  render() {
+    const { tokens } = this.props;
+    return (
+      <section className="token-balances">
+        <h5>{translate('sidebar_TokenBal')}</h5>
+        <table className="account-info">
+          <tbody>
+            {tokens
+              .filter(token => !token.balance.eq(0) || token.custom || this.state.showAllTokens)
```

Comment:
Ah, I was wondering where `.eq` was coming from but now I see that `token.balance` is from `Big.js` 👍 

This is all good, just leaving a note for anyone else reviewing ;)